daisy_workflows/image_import/windows/run_startup_scripts.cmd

```diff
@@ -16,12 +16,28 @@ REM limitations under the License.
 REM Give the network time to initialize.
 ping 127.0.0.1 -n 60
 
-echo "Translate: Starting image translate..." > COM1:
+set image_import_reg_path="HKLM\Software\ImageImport"
 
-REM Restart the system in 5 minutes. Translate.ps1 will cancel this restart when it starts.
+for /f "tokens=3" %%a in ('reg query %image_import_reg_path% /v StartupScriptRunsCount') do set startup_script_runs_count=%%a
```

Comment:
Extra comments here would be useful, describing what's the purpose of this block

Reply:
I updated my PR to use the filesystem instead of the registry (see this [discussion](https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1858#discussion_r812175449), please). I added some comments, however, I would be happy if you could have another look.

Reply:
Comments look good.